src/twisted/conch/ssh/userauth.py

```diff
@@ -623,13 +640,14 @@ def _cbGetPublicKey(self, publicKey):
         else:
             return False
 
-    def auth_password(self):
+    # start of auth methods: must all be (() -> bool), per
```

Comment:
It was not clear that this comment marks the start of a "section"

Maybe it helps to add some extra markup 

```suggestion
    # -----------------------
    # start of auth methods: must all be (() -> bool), per
```

Reply:
Personally this kind of arbitrary, inconsistent ASCII decoration just reads to me as messy.  Once you start down that road, why not `# ---~~~=== .oO{{{ BeGiNnInG oF aUtH MeThOdS }}}Oo. ===~~~---`.  I could try to be a bit clearer about this in the prose, though.